dev/src/Domain/Store.java

```diff
@@ -39,7 +39,7 @@ public void addItem(ProductInStore productInStore) {
         if (this.shelves.size() == 0) {
             this.shelves.add(new Shelf(new ItemStack(productInStore, 1)));
         } else {
-           this.shelves.getFirst().addItemStack(new ItemStack(productInStore, 1));
+           //this.shelves.getFirst().addItemStack(new ItemStack(productInStore, 1));
```

Comment:
This I commented since the function doesn't seems to exist, some one deleted it? (on purpose / accident)